docs/configuration/configuration.md

```diff
@@ -850,11 +850,25 @@ tls_config:
 namespaces:
   names:
     [ - <string> ]
+
+# Polling interval
+[ resync_period: <duration> | default = 10m ]
+
+# Optional label and filter selectors to limit the discovery process to a subset of available resources. 
+[ label_selector: <string> ]
+[ field_selector: <string> ]
 ```
 
 Where `<role>` must be `endpoints`, `service`, `pod`, `node`, or
 `ingress`.
 
+By default `label_selector` and `field_selector` are empty, so the discovery pulls all resources 
```

Comment:
This is a bit more wordy than it needs to be, see what we say in consul. It'd be better to link the docs by the fields.